README.md

```diff
@@ -1,222 +1,211 @@
+**`map-deep-get`** Gets the font-size of h4 elements from Foundation settings.
 
-Checking logs
-```tail -f tsg.log````
 
-## stuff worth checking
-- smacss.com
-- bem.info
-- [Foundation docs]
-- [sc5 docs]
-- [sc5 build options]
+By wrapping in smaller mixins, we allow the end user to choose if they want to include our entire component style definitions or only part of them.
 
-## Working with the styleguide
+Possibility to pass a variable with a default value `($font-size: $base-font-size)` makes the mixin more versatile.
 
-SC5-Styleguide, parses SCSS comments for a standalone component library.
 
-Currently existing styles, should be most up-to-date but this tool will not be maintained in the future:
-https://company-81847.frontify.com/d/VHfNy7brWRvI/tieto-style-guide#/introduction/overview
+ ```
+@mixin component-font-styles($font-size: $base-font-size)
+  @debug '$font-size is #{$font-size}';
+  font-size: $font-size;
+  margin-bottom: $global-margin;
+}
+```
+
+```
+@mixin my-component {
+  .my-component {
+    @include component-font-styles($component-font-size);
+    color: $component-color;
+  }
+}
+```
+
+Don't forget that Sass has a lot of built-in features making it more like a programming language than writing CSS. See https://sass-lang.com/documentation or http://thesassway.com.
 
-SC5-styleguide
-https://github.com/SC5/sc5-styleguide
+## Logs
 
-Built on KSS documenting syntax
-http://warpspire.com/kss/
+You can check logs on DEMO environment (locally, you can check the log in the terminal where you NPM started the styleguide) by navigating into the folder of styleguide and running command **`tail -f tsg.log`**.
 
-Build options
-https://github.com/SC5/sc5-styleguide#build-options
+Alternatively, you can check the whole log file through text editor of your choice (nano for example) or **`cat`** command.
 
-Check links in the Additional Info section of the github project. There are some useful things for advanced techniques.
+## Template for standalone
 
-### Template for standalone
+@TODO - no idea what was meant to be here, Liisa.
 
-@TODO
+## Build
+@TODO: Should we keep this if it does not work? What exactly is this Build anyway?
+To build the style guide type 'gulp styleguide' it will build the guide into the 'docs' folder. (DOESNT WORK)
 
-## Usage
-Include SCSS files from the folder 'styles-library'.
+## Watch
+(@TODO: refactor this section, need more info from Liisa/Eriks - what is this "Watch"? Elaborate please.)
 
-## Style Guide
-Style guide is generated by SC5 styleguide generator and its files reside in '/style-guide'.
+Check the gulpfile for tasks that can be run to automate watch. With these instructions, local changes will not always be visible without restart. [@TODO Liisa can fill this in]
 
-### Development
-To develop the style guide type 'npm start' it will open up a server on port localhost:3000. (WORKS)
+(@TODO: shouldn't this following text be elsewhere in the readme? In different section...)
```

Comment:
I was trying to separate "Using the styleguide" from "Developing the styleguide" and this falls into the category of "developing"